docs/source/api/apollo-server.md

```diff
@@ -343,15 +343,44 @@ addMockFunctionsToSchema({
    to standard error. Specify this function to process errors in a different
    way.
 
-*  `privateVariables`: Array<String> | boolean
+*  `privateVariables`: Array<String\> | boolean
```

Comment:
Backslash?

Reply:
Oh-- the  `<String>` part wasn't showing up in markdown so I escaped it, don't know if there's a better convention for this?

Reply:
Ah I think @trevorblades is the expert on exactly how our docs system works these days.

Reply:
@helenwh you can use the &gt; and &lt; HTML entities to render those characters

Reply:
Thanks @trevorblades :)